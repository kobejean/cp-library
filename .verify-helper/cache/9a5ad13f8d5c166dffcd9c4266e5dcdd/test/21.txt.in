48
466 405
247 563
904 411
553 500
140 570
450 276
799 343
430 679
662 210
463 945
185 611
140 533
497 140
156 368
667 931
833 627
620 165
136 303
628 870
577 187
577 883
876 821
934 293
319 569
497 506
847 567
78 461
770 302
72 755
901 543
508 381
349 664
476 554
729 118
825 854
388 798
295 409
306 719
195 755
938 422
460 978
544 607
144 471
368 244
18 472
913 427
128 819
839 747
