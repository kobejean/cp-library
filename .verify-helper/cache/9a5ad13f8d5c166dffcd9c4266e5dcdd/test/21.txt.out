489.001022493818513659
